torch/nested/_internal/nested_tensor.py

```diff
@@ -264,13 +262,6 @@ def __tensor_unflatten__(inner_tensors: Dict, meta, outer_size, outer_stride):
             metadata_cache["max_seqlen"] = max_seqlen_tensor
         ragged_idx = meta["ragged_idx"]
 
-        # Alternatively, we could make it the caller's responsibility to
```

Comment:
I don't see a PR descr, but am I correct in thinking that this PR is mainly cleanup on top of the lower PR in the stack?

Reply:
Ahh sorry, yes that is right. Also there should be more context in this comment https://github.com/pytorch/pytorch/pull/130292#discussion_r1698733743